app/stacks/global/front-door/waf-policy.tf

```diff
@@ -169,3 +169,188 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
 
   tags = var.common_tags
 }
+
+resource "azurerm_frontdoor_firewall_policy" "applications_frontend" {
+  name                              = replace("pinswafapplicationsfrontend${local.resource_suffix}", "-", "")

```

Comment:
Can we call this `appeals_frontend` instead?

Reply:
Oops yep we sure can. Not me forgetting which team we are 😅